frontend/ui/wikipedia.lua

```diff
@@ -768,9 +768,9 @@ time, abbr, sup {
 
     -- Done with adding files
     if cancelled then
-        progress_callback(_("Cleaning up..."))
+        progress_callback(_("Cleaning up…"))
```

Comment:
Thanks!
To clean any badly encoded stuff, could you please:
line 672: replace the 3 or 4 bad bytes at end with some underscores: Fran____oix, as I wrote next line what they are
line 684: remove the stuff after PageTitle : PageTitle`©on`</a> , as I probably forgot to clean that when I replaced a real page title with this dummy PageTitle.

Btw, in case you've been too focused on wikipedia.lua and haven't seen that, your Travis build fails because you made a `\n` into a `\I` in evernote.koplugin/main.lua.